nilearn/maskers/nifti_masker.py

```diff
@@ -47,18 +47,18 @@ def __call__(self, imgs):
         )
 
 
-def _get_mask_strategy(strategy):
+def _get_mask_strategy(strategy: str):
```

Comment:
we could possibly have a single _get_mask_strategy function for multi and non multi maskers but this is not an urgent refactoring